styles/style.css

```diff
@@ -0,0 +1,855 @@
+}
+
+.subscribe-title {
+  align-self: center;
+  text-align: center;
+  width: 90%;
+  z-index: 1;
+}
+
+.subscribe-title h3 {
+  font-size: 18px;
+  color: #000;
+  font-weight: 700;
+  margin-bottom: 20px;
+}
+
+.subscribe-title p {
+  line-height: 18px;
+  margin-bottom: 20px;
+  color: #333;
+  font-size: 13px;
+}
+
+.news-letter-info {
+  flex-direction: column;
+  align-items: center;
+}
+
+.news-letter-sign-up-title {
+  font-size: 19px;
+  font-weight: 700;
+  margin-bottom: 15px;
+}
+
+.sign-up-info {
+  color: #333;
+  font-size: 13px;
+  margin-bottom: 20px;
+  font-weight: 700;
+  text-align: center;
+  line-height: 18px;
+}
+
+.sign-up-newsletter {
+  max-width: 100%;
+  margin-bottom: 30px;
+  border: 1px solid #e5e5e5;
+  background-color: #f4f5f6;
+  padding: 15px;
+}
+
+.social-footer,
+.footer-logo {
+  flex-direction: column;
+}
+
+.social-footer {
+  border-bottom: 1px solid rgba(3, 3, 3, 0.12);
+  margin-bottom: 10px;
+}
+
+.footer-logo svg {
+  width: 236px;
+  fill: #f72210;
+  margin: 0 15px 12px 0;
+}
+
+.copyright {
+  font-size: 12px;
+  color: #666;
+  margin-bottom: 12px;
+  text-transform: uppercase;
+}
+
+.social-icons i {
+  font-size: 23px;
+  color: #666;
+  margin: 0 10px 10px 0;
+}
+
+.info-footer {
+  font-size: 14px;
+  color: #666;
+  margin-bottom: 30px;
+}
+
+.sign-up-img {
+  width: 48%;
+  max-width: 191px;
+  background-image: url(../images/starting5-homepage.png);
+  background-size: cover;
+  background-position: center;
+}
+
+.subscribe-img {
+  justify-content: center;
+}
+
+@media (max-width: 767px) {
+  .sign-up-img {
+    display: none;
+  }
+
+  h4 {
+    font-size: 18px;
+    margin-bottom: 10px;
+  }
+}
+
+@media (min-width: 768px) {
+  .hero-section-container {
+    display: grid;
+    grid-template-columns: repeat(3, minmax(0, 1fr));
+    gap: 10px;
+  }
+
+  .footer-logo {
+    flex-direction: row;
+    align-items: flex-end;
+  }
+
+  .social-footer {
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: flex-end;
+  }
+
+  .culture {
+    grid-column: span 3;
+  }
+
+  .heroe-img-category {
+    font-size: 10px;
+  }
+
+  .mag-cat {
+    font-size: 13px;
+  }
+
+  .subscribe-section {
+    border: none;
+    overflow: hidden;
+    margin-bottom: 80px;
+  }
+
+  .subscribe-title {
+    margin-top: 30px;
+    text-align: start;
+  }
+
+  .sign-up-newsletter {
+    padding: 0;
+  }
+
+  .sign-up-info {
+    margin: 15px;
+  }
+}
+
+@media (max-width: 1023px) {
+  nav,
+  .weather,
+  .nav-subscribe {
+    display: none;
+  }
+}
+
+@media (min-width: 1023px) {
+  .navbar-logo svg {
+    fill: white;
+    width: 396px;
+    height: 60px;
+  }
+
+  nav {
+    margin-top: 90px;
+  }
+
+  .menu-button,
+  .free-trial-button {
+    display: none;
+  }
+
+  .sign-up-container {
+    margin-top: 35px;
+  }
+
+  .subscribe-title {
+    max-width: 250px;
+  }
+}
+
+.last-bottom-pick-side li:first-child {
+  padding-top: 0;
+}
+
+/* helper */
+.flex { display: flex; }
```

Comment:
This could also be refined further by relying on bootstrap classes. ([Responsive-breakpoints](https://getbootstrap.com/docs/4.3/layout/overview/#responsive-breakpoints))